Resources/Prototypes/Recipes/Construction/Graphs/structures/station_ai_core.yml

```
@@ -0,0 +1,79 @@
+﻿- type: constructionGraph
+  id: StationAiCore
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: frame
+          steps:
+            - material: Plasteel
+              amount: 4
+              doAfter: 5
```

```suggestion
  graph:
  - node: start
    edges:
    - to: frame
      steps:
      - material: Plasteel
        amount: 4
        doAfter: 5
```
minor nitpick: the indentation doesn't follow conventions
same below